Resources/Prototypes/Entities/Structures/Machines/fax_machine.yml

```diff
@@ -4,6 +4,7 @@
   name: long range fax machine
   description: Bluespace technologies on the application of bureaucracy.
   components:
+  - type: StationAiWhitelist
```

Comment:
Oh, one thing to clean up:
Move it a little lower in the list, components are usually ordered by importance and grouped thematically.
So the Sprite should stay on top. I would suggest move `StationAiWhitelist` below `UserInterface`.

Reply:
Didn't know that, thanks. Changes applied.